src/Interfaces/MoesifHooksInterface.php

```diff
@@ -5,14 +5,14 @@
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 
-interface EventSubscriberInterface {
+interface MoesifHooksInterface {
     public function identifyUserId(Request $request, Response $response): ?string;
     public function identifyCompanyId(Request $request, Response $response): ?string;
     public function identifySessionToken(Request $request, Response $response): ?string;
     public function getMetadata(Request $request, Response $response): ?array;
     public function skip(Request $request, Response $response): bool;
     public function maskRequestHeaders(array $headers): array;
     public function maskResponseHeaders(array $headers): array;
-    public function maskRequestBody(string $body): ?string;
-    public function maskResponseBody(string $body): ?string;
-}
```

Comment:
this is because it can beither an String or Array (aka json object), but syntax of "string|array" is not supported before php 8. so i decided to remove any type decclartion to increase compatibility